docs/stepbystep.md

```diff
@@ -52,20 +52,20 @@ information interchanged between simulated devices and the Context Broker.
 
 The following list shows the prerequisite software and versions:
 
-* Orion Context Broker (v0.26)
+* Orion Context Broker (v1.0)
```

Comment:
Is the procedure not working with v0.26?

Reply:
This is to raise the discussion mentioned in #216. I'd keep it as it is in lack of responses, but since I was taking the chance to update this file...

I honestly haven't tested other versions than the latest, so who maybe know this is @dmoranj. I coudn't spot any hint of the correct version by looking in this repo, maybe a different one has been used for integration tests that I'm not aware of?

Reply:
Maybe a solution is to consider both, I mean:

```
* Orion Context Broker (we have tested this procedure with v0.26 and v1.0)
```

or a similar wording.

Reply:
I'll leave it as it was, the change would not add any value.

Reply:
NTC